scipy/spatial/transform/rotation.py

```diff
@@ -872,3 +874,31 @@ def __getitem__(self, indexer):
                   array.
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def random(cls, num=None, state=None):
+        """Generate rotations uniformly distributed in 3D space.
+
+        Parameters
+        ----------
+        num : None or int
+            Number of random rotations to generate. If `None`, then a single
```

Comment:
If None (default), ...

Also I suggest not to use single backticks for things like None, they are to reference arguments.